staff.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'highfive'
+class Staff
+
+  attr_reader :phase, :age, :name, :target_raise
+  def initialize(options={})
+    @phase = 3
+    @age = options.fetch(:age, 0)
+    @name = options.fetch(:name, "")
+    @target_raise = 1000
+  end
```

Comment:
This initialize method gets overridden by both subclasses.  Since they do a lot of the same things, use super to call into this from the subclasses, and only do the stuff specific to the subclass in the subclasss.

Same for the attr_readers (remove the common ones from the subclasses).